src/compiler/scala/tools/nsc/backend/jvm/BTypesFromSymbols.scala

```diff
@@ -149,6 +150,23 @@ abstract class BTypesFromSymbols[G <: Global](val global: G) extends BTypes {
     new asm.Handle(asm.Opcodes.H_INVOKESTATIC, ownerInternalName, sym.name.encoded, descriptor, isInterface)
   }
 
+  def handleFromMethodSymbol(sym: Symbol): asm.Handle = {
```

Comment:
Since there is an `isField` on `Symbol`, it would be quite nice to generate a `asm.Opcodes.H_GETFIELD` handle for those. I don't have a good code suggestion though (not sure how to unmangle the field name and its trailing whitespace)...

Reply:
I think I'll leave this to a followup improvement.